packages/uniforms-bridge-json-schema/__tests__/JSONSchemaBridge.ts

```diff
@@ -151,6 +151,23 @@ describe('JSONSchemaBridge', () => {
           },
         ],
       },
+      nonObjectAnyOfRequired: {
+        anyOf: [
+          {
+            const: 'alphabetic',
+            type: 'string',
+          },
+          {
+            enum: ['top', 'middle', 'bottom'],
+            type: 'string',
+          },
+          {
+            type: 'number',
+            minimum: 0,
+          },
+        ],
+        required: true,
```

Comment:
It shouldn't be here but in the `required` array two lines later, where the `dateOfBirth` is.

Reply:
Yes, and this mistake caused the mess with the `definition.required` not being an array. The new commit fixes it all. 
Thanks for the clarification.